wafer/talks/static/wafer/talks/talk-categorization.js

```diff
@@ -0,0 +1,30 @@
+var $ = (jQuery || django.jQuery)
+
+$(document).ready(function() {
+
+    function formatTalkCategorizationResult(item) {
+        var parts = item.text.split(/:\s*/)
+        var name = parts[0]
+        var description = parts[1]
+        return $([
+            "<div>",
+            "<div>",
+            "<strong>",
+            name,
+            "</strong>",
+            "</div>",
+            description,
+            "</div>"
+        ].join(""))
+    }
+
+    function formatTalkCategorizationSelection(item) {
+        return item.text.split(/:\s*/)[0]
+    }
+
+    $('.talk-categorization').select2({
+        templateResult: formatTalkCategorizationResult,
+        templateSelection: formatTalkCategorizationSelection,
+        dropdownAutoWidth: true
+    })
+})
```

Comment:
We already depend on django-easy-select2 for the schedule - isn't it possible to reuse that for this, rather than adding custom logic ?

Reply:
it's not.

I read the documentation for both easy-select2 and select2, and the relevant code from select2, and there is no way to pass any non-string data (such as those callback functions) othen than via the select2() call.

there were more than one attempt to add that to select2 (e.g. select2/select2#2957 and select2/select2#4702) but none of them got anywhere.